src/main/ngapp/src/app/login/login.component.ts

```diff
@@ -24,16 +24,6 @@ export class LoginComponent implements OnInit {
       .login(email, password)
       .then((res: any) => {
         localStorage.setItem('user', JSON.stringify(res.user));
-
-        // Calling firebase.auth() to get idToken that will be sent to the filter in the backend
-        // for authentication using FirebaseApp SDK. The idToken is stored in localStorage.
-        firebase.auth().currentUser.getIdToken()
-            .then((idToken) => {
-            localStorage.setItem('idToken', idToken);
-            console.log('idToken in login', localStorage.idToken);
-        });
-
-
```

Comment:
why is this being removed?

Reply:
I moved this to the auth service, where we set user in local storage as well

Reply:
alright